java/test/org/openqa/selenium/interactions/PenPointerTest.java

```diff
@@ -407,8 +407,8 @@ public void setPointerEventProperties() {
 
     Rectangle rect = pointerArea.getRect();
 
-    int centerX = (int) Math.floor(rect.width / 2 + rect.getX());
```

Comment:
in this case, I followed the rules of arithmetic operations on integers. When dividing an integer number `rect.width ` by an `int`, the result will be the integer part of the division - the same as what `Math.floor()` does, but without converting to a `double`. Therefore, the conversion to `int` after the parentheses is also unnecessary. It turns out that in this expression, only integer operands can be operated on if only the integer part of the division operation is required.